conf/app.routes

```diff
@@ -3,7 +3,8 @@
 ->         /                                                 app.identification.Routes
 ->         /                                                 app.locationOfGoods.Routes
 ->         /                                                 app.identification.authorisation.Routes
-->         /                                                 app.incident.Routes
```

Comment:
I think you've accidentally overwritten the app.incident.Routes

Reply:
Fixed